package/Dockerfile

```diff
@@ -1,3 +1,6 @@
-FROM alpine
+FROM registry.suse.com/suse/sle15:15.3
+RUN zypper -n update && \
```

Comment:
This shouldn't be necessary, we update the bci base image frequently.

Reply:
I added this just to be on the safe side. Twice I pulled an image and `zypper update` downloaded a few packages. Maybe it was related to that issue where the packages were not being properly refreshed? 🤷🏻‍♂️ 

Do you believe it's safe to remove it?